query-node/codegen/package.json

```diff
@@ -11,6 +11,6 @@
   },
   "dependencies": {
     "@joystream/hydra-cli": "5.0.0-alpha.4",
-    "@joystream/hydra-typegen": "5.0.0-alpha.4"
+    "@joystream/hydra-typegen": "file:../../joystream-hydra-typegen-v5.0.0-alpha.4.tgz"
```

Comment:
We want to use the newly published [5.0.0-alpha.6](https://www.npmjs.com/package/@joystream/hydra-typegen/v/5.0.0-alpha.6)

Reply:
Yeah, sorry, figured out that I didn't save the file before adding the commit